gateway/src/apicast/policy/3scale_batcher/3scale_batcher.lua

```diff
@@ -198,8 +208,12 @@ function _M:access(context)
   ensure_timer_task_created(self, service_id, backend)
 
   local cached_auth = self.auths_cache:get(transaction)
+  local auth_is_cached = (cached_auth and true) or false
+  metrics.update_cache_counters(auth_is_cached)
```

Comment:
Not sure if it would be any clearer, but I'd go for:

```lua
metrics.update_cache_counters(cached_auth)
```

And there:

```lua
function _M.update_cache_counters(cached_auth)
   func_update_counters[not not cached_auth]()
 end
```

No need to change anything though.

Reply:
Just a matter of preference over a small detail, but I prefer my solution because it does not have the `not not`.